types/primus/index.d.ts

```diff
@@ -192,5 +193,6 @@ declare namespace Primus {
         on(event: 'reconnect timeout' | 'reconnect failed', handler: (err: Error, opts: ReconnectEventOpts) => void): this;
         on(event: 'data', handler: (message: any) => void): this;
         on(event: 'error', handler: (err: Error) => void): this;
+    	open(): void;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/primus/index.d.ts;
  Line: 196;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.